docker-compose.yml

```diff
@@ -29,6 +29,8 @@ services:
     volumes:
       - .:/var/www/html
       - ./data/data:/data
+      - /var/www/html/mod
```

Comment:
This will disable live edits of mods ... the mods will be loaded from the docker image rather than mounted from a host volume.. you would need to rebuild the docker image each time changes are made to modules.